src/rail/estimation/algos/deepdisc.py

```diff
@@ -143,11 +145,41 @@ class DeepDiscInformer(CatInformer):
     def __init__(self, args, comm=None):
         CatInformer.__init__(self, args, comm=comm)
 
+    def inform(self, training_data, metadata):
+        self.set_data('input', training_data)
+        self.set_data('metadata', metadata)
+        self.run()
+        self.finalize()
+        return self.get_handle('model')
+
+    def finalize(self):
+        self.temp_dir.cleanup()
+
     def run(self):
         """
         Train a inception NN on a fraction of the training data
         """
         train_data = self.get_data("input")
+        metadata = self.get_data("metadata")
+
+        # create an iterator here
+        itr = self.input_iterator("input")
+        self.temp_dir = tempfile.TemporaryDirectory()
+        for start_idx, _, chunk in itr:
+            for idx, image in enumerate(chunk):
+                this_img_metadata = metadata[start_idx + idx]
+                height = this_img_metadata["height"]
+                width = this_img_metadata["width"]
+
+                #! This spot could have bugs width, height, or height, width???
+                reformed_image = image.reshape(6, width, height).astype(np.float32)
```

Comment:
The `6` is LSST-specific, and refers to the number of filters used. We should probably make this a top level variable, `NUM_LSST_FILTERS = 6`, so that it's clear what this number represents.

Reply:
We should really make this a user-specified input, since future datasets may have different channels

Reply:
Tracked in https://github.com/lincc-frameworks/deepdisc/issues/75

Reply:
Referenced in issue #23